docs/content/end-users/configuration.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Create a Porter Config File
+description: Learn how to customize Porter's default behavior through configuration file
+---
+
+Porter's default behavior, such as log level and default plugins, etc., can be modified through its config file.
+The config file also provides the ability to define plugin specific
+configurations, so you don't need to repeat them each time when you run porter.
+
+## Location
+
+The configuration file should be stored in the PORTER_HOME directory, by default
+~/.porter/. In the example below, we are going to use the yaml format, however,
+porter supports other file formats for its config file as well. For more detailed
+information, please see [configuration](/configuration/#config-file).
+
+First check if you have PORTER_HOME set on your machine:
+```console
+$ echo $PORTER_HOME
```

Comment:
The porter documentation should always show commands that work in any shell (like porter commands) or we need to provide an example for each shell (Bash + Powershell).

I think the docs here could be redone pretty easily to avoid "bash-isms" by not setting the PORTER_HOME env var (which I honestly don't recommend anyone should set anyway unless they really need to change the home directory which we aren't doing here). Then you can reference were to locate the config file with "in your PORTER_HOME directory, which defaults to ~/.porter" and leave it up to them to find the directory.